### PR TITLE
fix:TOS-812 The Element creation introduction vedio is not visible and also the overlay is not scrollable in Mobile Inspector

### PR DESCRIPTION
**Pull request by manikanta-reddy-t**

### CODE REVIEW

File: ui/src/scss/_sizing.scss [hunk 1]
```diff
@@ -186,6 +186,9 @@ $abbrev in (width: w, height: h) {
 .min-w-250p {
   min-width: 250px;
 }
+.h-100-60{
```
Comment (guganabu) [inline ui/src/scss/_sizing.scss:189]: use rem value, limit the usage of px units
h-100-60 class name can be rephrase to convey better

Comment (manikanta-reddy-t) [reply ui/src/scss/_sizing.scss]: done used rem and rephrased h-100-n60

File: ui/src/scss/_sizing.scss [hunk 2]
```diff
@@ -186,6 +186,9 @@ $abbrev in (width: w, height: h) {
 .min-w-250p {
   min-width: 250px;
 }
+.h-100-n60{
```
Comment (guganabu) [inline ui/src/scss/_sizing.scss:189]: what does it mean n-60 and it seems, the class is nowhere used @manikanta-reddy-t

Comment (manikanta-reddy-t) [reply ui/src/scss/_sizing.scss:189]: n-60 means negative 60

Comment (manikanta-reddy-t) [reply ui/src/scss/_sizing.scss:189]: used in latest commit,forgot to add in earlier commit

**Review by rajesh-testsigma — APPROVED** [review]

**Pull request merged**